metricbeat/module/ceph/df/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- name: df
+  type: group
+  description: >
+    df
+  fields:
+    - name: example
```

Comment:
Fields docs have to be added. This will also make it easier for us to understand what kind of data is provided.